osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerSpunOut.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Rulesets.Osu.Mods;
+
+namespace osu.Game.Rulesets.Osu.Tests
+{
+    [TestFixture]
+    public class TestSceneSpinnerSpunOut : TestSceneSpinner
+    {
+        public override IReadOnlyList<Type> RequiredTypes => base.RequiredTypes.Concat(new[] { typeof(OsuModSpunOut) }).ToList();
+
+        [SetUp]
+        public void SetUp() => Schedule(() =>
+        {
+            SelectedMods.Value = new[] { new OsuModSpunOut() };
```

Comment:
Spinner automatically spins in game but does not in this test.

Reply:
It doesn't spin because `frameDelay` is always 0 inside the mod implementation. I assume you need an actual backing clock to test this properly.

Reply:
Fixed by 596f4f7d2e6c62c7a4c3fa43ce161c5b71c388d8